lib/python/gladevcp/combi_dro.py

```diff
@@ -424,18 +427,18 @@ def _position(self, object, p, rel_p, dtg, joint_actual_position):
         # joint_actual_position = joint positions, not needed here
         
         try:
-            dtg = dtg[self.axis_no]
-            abs_pos = p[self.axis_no]
-            rel_pos = rel_p[self.axis_no]
+            self.dtg = dtg[self.axis_no]
+            self.abs_pos = p[self.axis_no]
+            self.rel_pos = rel_p[self.axis_no]
```

Comment:
I can also take the scale factor at this position into account. Is maybe more logical here.

```suggestion
            dtg = dtg[self.axis_no]
            abs_pos = p[self.axis_no]
            rel_pos = rel_p[self.axis_no]
```            
```  
...
        if self.diameter:
            scale = 2.0
        else:
            scale = 1.0
...            
```
```diff
+           self.dtg = dtg * scale
+           self.abs_pos = abs_pos * scale
+           self.rel_pos = rel_pos * scale
```

Reply:
I have implemented this. Much better i think.